lib/manager/gradle-lite/readme.md

```diff
@@ -18,3 +18,25 @@ To disable the original `gradle` manager if `gradle-lite` satisfies all requirem
 ```
 
 Feedback for this manager would be welcome as we would like to switch it to be the default manager for Gradle soon.
+
+`gradle-lite` supports the following version definition strategies:
+
+- Version defined as a plain string in a `*.gradle` file, for example: `"org.springframework.boot:spring-boot-starter-web:2.5.1"`
+
+- Version defined as a local variable:
+
+  - In `build.gradle`:
+
+    ```
+    def springBoot='2.5.1'
+    ...
+    "org.springframework.boot:spring-boot-starter-web:$springBoot"
+    ```
+
+- Version defined as a variable in the accompanying `*.properties` file:
+  - In `gradle.properties`: `springBoot=2.5.1`
+  - In `build.gradle`: `"org.springframework.boot:spring-boot-starter-web:$springBoot"`
+
+Variables defined in other custom locations are not currently supported.
```

Comment:
```suggestion
Variables defined in other custom locations are not supported.
```

I like this better, as the new sentence only states "no support", instead of "might get supported later on"

Reply:
Agreed. It's funny because I ended up adding "currently" for lack of a better way to say that I didn't want to imply anything about future support... But as is often the case, simpler is better ;-)